core/src/main/java/edu/wpi/grip/core/operations/network/networktables/NTManager.java

```diff
@@ -39,17 +46,14 @@
  */
 @Singleton
 public class NTManager implements Manager, MapNetworkPublisherFactory, MapNetworkReceiverFactory {
-  /*
-   * Nasty hack that is unavoidable because of how NetworkTables works.
-   */
-  private static final AtomicInteger count = new AtomicInteger(0);
```

Comment:
I forget why this was required. Can you remind me why we had it?

Reply:
Basically, just tracking usages of the NetworkTables connection. The client would only run if there was at least one source or one publishing operation that used it - if not, then the client would be shut down. I believe this was to prevent GRIP from keeping old entries around, but ntcore makes it so that clients only keep around entries that they have written to (the old library would make the client keep around _all_ entries and re-send them to a server on a reconnect).